tests/storage/storage.py

```diff
@@ -1,14 +1,13 @@
-from lib.common import wait_for
-import subprocess
+from lib.common import SSHCommandFailed, wait_for
 
 def try_to_create_sr_with_missing_device(sr_type, label, host):
     try:
         sr = host.sr_create(sr_type, label, {}, verify=True)
-    except subprocess.CalledProcessError as e:
+    except SSHCommandFailed as e:
         assert e.stdout == (
-            b'Error code: SR_BACKEND_FAILURE_90\nError parameters: , '
-            b'The request is missing the device parameter, \n'
-        ), 'Bad error, current: {}'.format(e.stdout)
+            'Error code: SR_BACKEND_FAILURE_90\nError parameters: , '
+            + 'The request is missing the device parameter, \n'
+        )
```

Comment:
This change (removal of useless assert error message) should be mentioned in the commit message.